webv2/primarykey/primarykey.go

```diff
@@ -0,0 +1,244 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package web defines web APIs to be used with harbourbridge frontend.
+// Apart from schema conversion, this package involves API to update
+// converted schema.
+
+package primarykey
+
+import (
+	"fmt"
+
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+	"github.com/cloudspannerecosystem/harbourbridge/webv2/session"
+)
+
+// updateprimaryKey updates primary key desc and order for primaryKey.
+func updatePrimaryKey(pkRequest PrimaryKeyRequest, spannerTable ddl.CreateTable) ddl.CreateTable {
+
+	for i := 0; i < len(pkRequest.Columns); i++ {
+
+		for j := 0; j < len(spannerTable.Pks); j++ {
+
+			id := getColumnId(spannerTable, spannerTable.Pks[j].Col)
+
+			if pkRequest.Columns[i].ColumnId == id {
+
+				spannerTable.Pks[j].Desc = pkRequest.Columns[i].Desc
+				spannerTable.Pks[j].Order = pkRequest.Columns[i].Order
+			}
+
+		}
+	}
+
+	spannerTable = insertOrRemovePrimarykey(pkRequest, spannerTable)
+	return spannerTable
+}
+
+// insertOrRemovePrimarykey performs insert or remove primary key operation based on
+// difference of two pkRequest and spannerTable.Pks.
+func insertOrRemovePrimarykey(pkRequest PrimaryKeyRequest, spannerTable ddl.CreateTable) ddl.CreateTable {
+
+	cidRequestList := getColumnIdListFromPrimaryKeyRequest(pkRequest)
+	cidSpannerTableList := getColumnIdListOfSpannerTablePrimaryKey(spannerTable)
+
+	//primary key Id only presnt in pkeyrequest
```

Comment:
In the comments follow the following pattern :
1. Give a space after // and start with capital letter
2. End the comment with a full stop